services/core-api/app/api/notice_of_departure/models/notice_of_departure.py

```diff
@@ -74,6 +75,10 @@ class NoticeOfDeparture(SoftDeleteMixin, AuditMixin, Base):
         primaryjoin=
         "and_(NoticeOfDeparture.nod_guid==NoticeOfDepartureDocumentXref.nod_guid, NoticeOfDepartureDocumentXref.deleted_ind==False)",
         order_by='desc(NoticeOfDepartureDocumentXref.create_timestamp)')
+    contacts = db.relationship(
+        'NoticeOfDepartureContact',
+        lazy='select',
+        primaryjoin="and_(NoticeOfDeparture.nod_guid==NoticeOfDepartureContact.notice_of_departure_guid, NoticeOfDepartureContact.deleted_ind==False)", )
```

Comment:
nice, we are lazyloading. I think we will need to indicate joining contacts when `find_one` method is called

Reply:
Yeah, this is the part I'm looking at now.  We can chat about it.